notebook.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    titles=[r"$\|| \nabla\mathbf{w}_0\||_2$",r"$\|| \nabla . \mathbf{w}_0\||_2$",'$\||\Delta \mathbf{w}_0\||_2$']
</pre><p>The order of norm <code>||</code>  and absolute <code>|</code> values in the figure titles does not match</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/reproduce-deep-prior-4Dvar/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Fixed. Thank you!</p>

Reply:
<p>closing conversation, resolved</p>